sources/Application/Views/InstrumentView.cpp

```diff
@@ -173,6 +173,11 @@ void InstrumentView::fillSampleParameters() {
   v = instrument->FindVariable(SIP_LOOPMODE);
   f1 = new UIIntVarField(position, *v, "loop mode: %s", 0, SILM_LAST - 1, 1, 1);
   T_SimpleList<UIField>::Insert(f1);
+  position._y += 1;
+
+  v = instrument->FindVariable(SIP_SLICES) ;
+  f1 = new UIIntVarField(position, *v, "slices: %2.2X", 1, 0xFF, 1, 0x10) ;
```

Comment:
Does it make sense to select slices in hex?
Does it make sense that the maximum is 255 slices in a sample? seems like a lot, maybe max would be number of midi notes? (128)

Reply:
Yeah I agree on both points =)